public_html/wp-content/plugins/pattern-creator/src/components/submission-modal/index.js

```diff
@@ -92,6 +92,61 @@ export default function SubmissionModal( { onClose, onSubmit, status } ) {
 	const hasError = notice && notice.status === 'error';
 
 	const pages = [
+		{
+			header: __( 'Publish your pattern', 'wporg-patterns' ),
+			content: (
+				<div>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'<a>Check the guidelines</a> for how to be successful when submitting your pattern.',
+								'wporg-patterns'
```

Comment:
I'm not a copywriter, but this one feels a bit too passive to me. 
Here's my attempt, take it or leave it:
"Check the guidelines for successfully submitting your pattern."